src/components/atoms/ProblemSubtitle/styles.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+
+import { SubtitleHeaderStyleProps } from "./interface";
+import Wrapper from "../Wrapper";
+import { minDevice } from "../../../assets/styles/breakpoints";
+import { blue200, placeholder } from "../../../assets/styles/colors";
+
+const SubtitleWrapper = styled(Wrapper)`
+  color: ${placeholder};
```

Comment:
I'd rename this color, similar to what @bazelajoanna did with blue200 and so on). Now seeing its usage we see the name should be different, as it looks a bit odd here. ;)

Reply:
Done :)